flink-runtime-web/web-dashboard/src/app/pages/job/overview/chart/job-overview-drawer-chart.component.html

```diff
@@ -26,14 +26,16 @@
       nzShowSearch
       [nzPlaceHolder]="'Select Metric Name'"
       (ngModelChange)="updateMetric($event)"
+      nzDropdownMatchSelectWidth="false"
+      [nzDropdownStyle]="{ 'min-width': dropdownWidth + 'px', 'max-width': '80vw' }"
```

Comment:
How has priority here, min or max width? Basically, is there a chance we overflow the parent component or are we always going to use at most 80% of the view port of the parent component? 

Could you share a small video where you add some metrics and resize the screen, making it really small and really big so we see how it behaves?

Reply:
@gustavodemorais we are going to use 80% of the view port of the **parent component** at max. For all the other ranges : metric names which are medium large to small upon selection and changes in the list the drop down component width would change. And based upon current selection the **Add metric** component width would also get adjusted.
[https://drive.google.com/file/d/13ONAlsQ-XeW5_0SSiA_x3H7aPGVJAVX2/view?usp=sharing](Varying metric lengths)
[https://drive.google.com/file/d/12nV_gk2Df6awtlTGc_P-P4GEAWI9G37y/view?usp=sharing](Varying Screen Size)

Reply:
@gustavodemorais  Could you review this , and let me know if this looks good to you?  Thanks!

Reply:
@davidradl Could you also review this PR, and let me know if this looks good to you. Thanks!